Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,24 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+const formatCurrency = (total) => (total / 100).toFixed(2);
+
+function formatReceiptDetails(str1, pad1Num, str2, pad2Num, str3) {
```

Comment:
Every place you call this, you pass the same values for the pads - I would argue these probably shouldn't be parameters to the function, but should probably be constants that the function always uses?

(If you were using this function to format different styles of receipts, you may have this extra-parametrisable function, but you'd probably have a wrapper function setting the values consistently for each style of receipt you were formatting)

Reply:
Yes, I agree. The values are the same, therefore would make sense to set them as constants